packages/gitbeaker-core/src/services/Issues.ts

```diff
@@ -69,6 +75,19 @@ export class Issues extends BaseService {
       ...options,
     });
   }
+  
+  deleteLink(
```

Comment:
Just for consistency, Use `remove` instead of `delete`

Reply:
Fixed